From 0e066792b58bf963852c8471ba32b54d759389c8 Mon Sep 17 00:00:00 2001
From: Juan Oxoby <juan@vicarious.com>
Date: Fri, 4 Sep 2020 14:26:08 -0700
Subject: [PATCH] Use PUBLIC for CPPZMQ::CPPZMQ

---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index f9f50c26..97772404 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -25,9 +25,9 @@ target_link_libraries(${PROJECT_LIBRARY_TARGET_NAME}
   PUBLIC
     ignition-cmake${IGN_CMAKE_VER}::utilities
     ignition-msgs${IGN_MSGS_VER}::ignition-msgs${IGN_MSGS_VER}
+    CPPZMQ::CPPZMQ
   PRIVATE
     ${ZeroMQ_TARGET}
-    CPPZMQ::CPPZMQ
 )
 
 # Windows system library provides UUID
-- 
2.17.1

