From a34796ddbd02d7df6dfda2c35ed30a0967970b41 Mon Sep 17 00:00:00 2001
From: Juan Oxoby <juan@vicarious.com>
Date: Thu, 3 Sep 2020 23:14:15 -0700
Subject: [PATCH] cmake fixes

---
 CMakeLists.txt     | 1 +
 src/CMakeLists.txt | 5 +++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0b934f9..b1a12dc7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -292,6 +292,7 @@ else (build_errors)
   set(sdf_version_output "cmake/${sdf_target}-config-version.cmake")
   set(sdf_config_install_dir "${LIB_INSTALL_DIR}/cmake/${PROJECT_NAME_LOWER}/")
 
+  include(CMakePackageConfigHelpers)
   #--------------------------------------
   # Configure and install the config file
   configure_package_config_file(
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c52c3200..8e87cbd6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -162,11 +162,12 @@ if (NOT WIN32)
 endif()
 
 sdf_add_library(${sdf_target} ${sources})
-target_link_libraries(${sdf_target} PUBLIC ${IGNITION-MATH_LIBRARIES})
+target_link_libraries(${sdf_target}
+  PUBLIC
+    ignition-math${IGN_MATH_VER}::ignition-math${IGN_MATH_VER})
 
 target_include_directories(${sdf_target}
   PUBLIC
-    ${IGNITION-MATH_INCLUDE_DIRS}
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:${INCLUDE_INSTALL_DIR}/..>
 )
-- 
2.17.1

