From e0e034051e86f57f40b26f270c77a2eb09ba055d Mon Sep 17 00:00:00 2001
From: Juan Oxoby <juan@vicarious.com>
Date: Thu, 3 Sep 2020 19:33:48 -0700
Subject: [PATCH] Add #define TIXML_USE_STL

---
 urdf_parser/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/urdf_parser/CMakeLists.txt b/urdf_parser/CMakeLists.txt
index 3952064..e0fda82 100644
--- a/urdf_parser/CMakeLists.txt
+++ b/urdf_parser/CMakeLists.txt
@@ -1,3 +1,5 @@
+add_compile_definitions(TIXML_USE_STL)
+
 include_directories(include)
 
 add_library(urdfdom_world SHARED src/pose.cpp src/model.cpp src/link.cpp src/joint.cpp src/world.cpp)
-- 
2.17.1

