From 460fd63ac56260868a10d0c57a816c150f0ef362 Mon Sep 17 00:00:00 2001
From: Juan Oxoby <juan@vicarious.com>
Date: Wed, 2 Sep 2020 14:09:53 -0700
Subject: [PATCH] Don't use IgnProtobuf

---
 CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 62b56b1..1bb5f43 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,11 +30,7 @@ message(STATUS "\n\n-- ====== Finding Dependencies ======")
 #--------------------------------------
 # Find Protobuf
 set(REQ_PROTOBUF_VER 3)
-ign_find_package(IgnProtobuf
-                 VERSION ${REQ_PROTOBUF_VER}
-                 REQUIRED
-                 COMPONENTS all
-                 PRETTY Protobuf)
+find_package(Protobuf ${REQ_PROTOBUF_VER} REQUIRED CONFIG)
 
 #--------------------------------------
 # Find ignition-math
-- 
2.17.1

